debs/focal/fits/Makefile

```diff
@@ -0,0 +1,51 @@
+NAME          = fits
+VERSION       ?= 1.1.0
+RELEASE	      ?= 6~20.04
+GIT_URL	      = https://github.com/harvard-lts/fits
+GPG_ID        ?= 5236CA08
+DEB_TOPDIR    = "/debbuild"
+DOCKER_VOLUME = "/src"
+DOCKER_IMAGE  = "debbuild-$(NAME)-$(VERSION)"
+
+.PHONY: build-docker-image build deb-build deb-clean deb-test update-changelog
+
+all: build-docker-image build
+
+build-docker-image:
+	@echo "==> Building Docker image with build environment."
+	@docker build --tag "$(DOCKER_IMAGE)" --build-arg GIT_URL="$(GIT_URL)" --build-arg GIT_BRANCH="$(VERSION)" --build-arg PACKAGE="$(NAME)" .
+
+update-changelog:
+	@echo "==> Update changelog."
+	@dch -v $(VERSION)-$(RELEASE) -D xenial "New upstream release"
```

Comment:
Should these couple of occurrences of `xenial` and the `trusty` below be updated to `focal`?

Reply:
Changed on new commits. Thanks!